tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -130,6 +131,11 @@ public function validInputProvider() {
 				'+2000000000000000-00-00T00:00:00Z',
 				TimeValue::PRECISION_Ga,
 			),
+			'-2000000000000000-00-00T00:00:00Z' => array(
```

Comment:
I'm not sure if it's a good idea to mark dates millions of years before BC as Julian. There should be a lower limit.